src/cookbooks/data_modeling/where_to_store.md

```diff
@@ -0,0 +1,47 @@
+# Introduction
+
+Mozilla’s current setup allows for the implementation of data modeling and business logic in different locations e.g BigQuery or Looker. The purpose of this document is to provide guidelines to decide where to store a new asset and to review pull requests that implement them.
+
+For the purpose of this documentation, the analytics data model is defined as the set of data assets designed to collect meaningful data from our raw datasets and structure it for an efficient understanding and analysis of our products, business processes and events.
+
+## What to check in a Pull Request in Looker
+
+When reviewing a PR first check if the data required is already implemented in an existing asset ( fully or partially) and prefer extending and existing dataset, over creating a new one or moving the calculation to Looker.
```

Comment:
Some of this content should probably live in https://github.com/mozilla/bigquery-etl/blob/main/.github/pull_request_template.md

Reply:
Yes, thank you, I agree the PR template is a better location for the PR review guidelines.

I added 2 follow-up PRs: one to update the [bigquery-etl PR template](https://github.com/mozilla/bigquery-etl/pull/3296) and one for a new [looker-spoke-default PR template](https://github.com/mozilla/looker-spoke-default/pull/435).
+ some rephrasing for a simpler language in the template.